packages/block-editor/src/components/inserter/tabs.js

```diff
@@ -14,12 +14,12 @@ const blocksTab = {
 const patternsTab = {
 	name: 'patterns',
 	/* translators: Patterns tab title in the block inserter. */
-	title: __( 'Patterns' ),
+	title: __( 'Imported Patterns' ),
```

Comment:
Yeh, that was a temporary naming - it will include theme and directory patterns I think, so `Directory` may not be the best name.

Reply:
Ideally it would just be called Patterns (or maybe Library in future) and we'd have a way to filter out directory patterns. The inserter in that way is basically a shortcut into your library but just like in your library we should have a way to explore patterns created by others (similar to installing block plugins).

In the short term, how challenging would it be for "Patterns" to also include user created "patterns"? The reusable block tab remains just synced patterns.

Reply:
> In the short term, how challenging would it be for "Patterns" to also include user created "patterns"? The reusable block tab remains just synced patterns.

This will be a little messy to implement, plus very confusing for users as they then have to learn that some patterns go one place and some another depending on whether they are synced or not, plus there patterns may have the same name as the imported ones so again potential confusion.

What if we have just have the Patterns tab, but at the top of that is a toggle to show theme/dir patterns or patterns they have added - we just need to come up with suitable labels to differentiate those two groups?

Reply:
I think we should handle this in a separate PR though, so will just move the position and naming back on this one tomorrow and we can follow that up once we get this one signed off and merged.

Reply:
Have added a note about the inserter panel rework to [this tracking issue](https://github.com/WordPress/gutenberg/issues/50159).